vulcanai2/models/basenetwork.py

```diff
@@ -131,10 +140,15 @@ def forward(self, inputs):
             outputs = []
             for model, x in zip(models, inputs):
                 outputs.append(model(x))
```

Comment:
need to test if each output shape gets independently preserved in the outputs list